### ISSUE TITLE
Build on Windows 10 and Many MAC versions Fails on rc4

### ISSUE THREAD

**Issue by Todd-Werelius** [angular/material#4791]
When running `gulp build` on windows 10 I get the following error

[18:28:50] Using gulpfile \Projects\angular#material\gulpfile.js
[18:28:50] Starting 'build-scss'...
[18:28:50] Building css files...
[18:28:50] Starting 'build-js'...
[18:28:50] building js files...
[18:28:54] Plumber found unhandled error:
 Error in plugin 'gulp-uglify'
Message:
    \Projects\angular#material\dist\angular-material.js: Unexpected token punc ((;)), expected punc ((,)) 
Details:
    fileName: \Projects\angular#material\dist\angular-material.js
    lineNumber: 22363
[18:28:54] Finished 'build-js' after 3.79 s
[18:28:54] Finished 'build-scss' after 3.83 s
[18:28:54] Starting 'build'...
[18:28:54] Finished 'build' after 18 #s

Basically it's at the end of the file and I see nothing wrong with it, though it's getting the line count wrong where the error is occurring for some reason, the file is only 22198 lines long according to my editor. 

Versions
OS : Windows 10
Gulp : 3.9.0 
gulp-uglify : 1.4.1

**Comment by hardywu** [comment]
Same here on Mac OS X

**Comment by doppio** [comment]
Same - I'm on OS X 10.10.4 and am unable to build.

**Comment by stephenpatten** [comment]
Ditto issue:

Mac OS X 10.11

**Comment by Todd-Werelius** [comment]
Just checked this against node 5.0, same problem, and the gulp-sass had to be updated to 2.1.0 in the package.json file to even get to that point.

**Comment by EladBezalel** [comment]
Working on my Windows 10, 

The problem is the version of node your'e using, unfortunately ngMaterial depends on gulp-sass version that doesn't support node > 12,
there's a PR #5045 about that.

**Comment by Todd-Werelius** [comment]
@EladBezalel I already overcame that, the problem is only in a few components where there is something funny going on in the directives sass, trying to track it down now.

**Comment by Todd-Werelius** [comment]
@EladBezalel

Okay got it working, just had to update  NPM and it's all gold by mostly following [PR #5045 fix](https://github.com/angular/material/pull/5045)

Described steps in that issue to fix

**Comment by alirezamirian** [comment]
+1
solution in #5045 fixes the problem
